armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -661,6 +668,7 @@ EOF
 
 # NGINX ------------------------------------------------------------------------
 apt install -y nginx
+rm /etc/nginx/sites-enabled/default || true
```

Comment:
By using `rm -f`, the command would exit success (`0`) even if the target file did not exist, and we could avoid the `|| true`.